### PR TITLE
add zh trans content/zh/docs/reference/setup-tools/kubeadm/generated/…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_apiserver-kubelet-client.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+<!--
+Renewals run unconditionally, regardless of certificate expiration date; extra attributes such as SANs will be based on the existing file/certificates, there is no need to resupply them.
+-->
+无论证书的到期日期如何，续订都会无条件地进行；SAN 等额外属性将基于现有文件/证书，因此无需重新提供它们。
+
+<!--
+Renewal by default tries to use the certificate authority in the local PKI managed by kubeadm; as alternative it is possible to use K8s certificate API for certificate renewal, or as a last option, to generate a CSR request.
+-->
+默认情况下，续订尝试在 kubeadm 管理的本地 PKI 中使用证书颁发机构；作为替代方案，可以使用 K8s 证书 API 进行证书更新，或者作为最后一个选择来生成 CSR 请求。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_apiserver-kubelet-client.md]:
```suggestion
默认情况下，续订尝试使用位于 kubeadm 所管理的本地 PKI 中的证书颁发机构；作为替代方案，也可以使用 K8s 证书 API 进行证书续订；亦或者，作为最后一个选择，生成 CSR 请求。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_front-proxy-client.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+<!--
+Renewals run unconditionally, regardless of certificate expiration date; extra attributes such as SANs will be based on the existing file/certificates, there is no need to resupply them.
+-->
+无论证书的到期日期如何，续订都会无条件地进行；SAN 等额外属性将基于现有文件/证书，因此无需重新提供它们。
+
+<!--
+Renewal by default tries to use the certificate authority in the local PKI managed by kubeadm; as alternative it is possible to use K8s certificate API for certificate renewal, or as a last option, to generate a CSR request.
+-->
+默认情况下，续订尝试在 kubeadm 管理的本地 PKI 中使用证书颁发机构；作为替代方案，可以使用 K8s 证书 API 进行证书更新，或者作为最后一个选择来生成 CSR 请求。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_front-proxy-client.md]:
```suggestion
默认情况下，续订尝试使用位于 kubeadm 所管理的本地 PKI 中的证书颁发机构；作为替代方案，也可能使用 K8s 证书 API 进行证书续订；亦或者，作为最后一种方案，生成 CSR 请求。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_selfhosting_pivot.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+<!--
+Convert static Pod files for control plane components into self-hosted DaemonSets configured via the Kubernetes API.
+-->
+将用于控制平面组件的静态 Pod 文件转换为通过 Kubernetes API 配置的自托管 DaemonSet。
+
+<!--
+See the documentation for self-hosting limitations.
+-->
+有关自我托管限制，请参阅文档。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_selfhosting_pivot.md]:
```suggestion
有关自托管的限制，请参阅相关文档。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_selfhosting_pivot.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+    <tr>
+      <td colspan="2">-f, --force</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      Pivot the cluster without prompting for confirmation
+      -->
+      在不提示确认的情况下 pivot 集群
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_selfhosting_pivot.md]:
```suggestion
      在不提示确认的情况下调整集群
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_selfhosting_pivot.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+    <tr>
+      <td colspan="2">--rootfs string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      [EXPERIMENTAL] The path to the 'real' host root filesystem.
+      -->
+      [实验] 指向 '真实' 宿主机的根目录。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_selfhosting_pivot.md]: 不准确

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_selfhosting_pivot.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+
+<!--
+SEE ALSO
+-->
+查看其他
+
+<!--
+* [kubeadm alpha selfhosting](kubeadm_alpha_selfhosting.md)	 - Make a kubeadm cluster self-hosted
+-->
+* [kubeadm alpha selfhosting](kubeadm_alpha_selfhosting.md)	 - 使 kubeadm 集群自托管
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_selfhosting_pivot.md]:
```suggestion
* [kubeadm alpha selfhosting](kubeadm_alpha_selfhosting.md)	 - 将 kubeadm 集群转换为自托管集群
```

**Comment by tanjunchen** [comment]
@tengqm  已经修改

**Pull request merged**